examples/snapshot/multi_zone_snap.py

```diff
@@ -0,0 +1,82 @@
+"""Sample script to demonstrate use of SoCo snapshot on multiple zones
+
+To use this script just run it - no configuration required.
+It shows it's capabilities better if you have something playing. 
+
+This is useful for scenarios such as when you want to switch to radio,
+an announcement or doorbell sound and then back to what was playing previously.
+
+This script has a play_alert function that will:
+ - take a snapshot of the current status of all sonos players (whats playing, 
+   volume etc.)
+ - play an alert_uri file on each group coordinator
+ - re-instate the sonos players to it's previous state
+ 
+This script does not group or un-group any Sonos players. Group management is
+a separate subject. For alerts grouping causes delays, so is avoided here.
+
+Note: the snap function is designed to be used once, Having taken a snap to 
+take another re-instantiate the class.
+"""
+
+import time
+import soco
+from soco.snapshot import Snapshot
+
+
+def play_alert(alert_uri, alert_volume=20, alert_duration=0, fade_back=False):
+    """
+    Demo function using soco.snapshot accrss multiple Sonos players.
+    
+    :param alert_uri: file that Sonos can play as an alert
```

Comment:
Consider using the Napoleon style for sphinc docstrings, which we are trying to switch to in general. Let me know if you need help.